src/main/scala/com/microsoft/hyperspace/index/serde/LogicalPlanSerDeUtils.scala

```diff
@@ -215,4 +221,22 @@ object LogicalPlanSerDeUtils {
             e.query.childOutputs))
     }
   }
+
+  object ExtractFileFormat {
+    def unapply(fileFormat: FileFormat): Option[FileFormat] = fileFormat match {
+      case CSVFileFormatWrapper => Some(new CSVFileFormat)
+      case JsonFileFormatWrapper => Some(new JsonFileFormat)
+      case _: ParquetFileFormat | _: OrcFileFormat => Some(fileFormat)
+      case f => throw HyperspaceException(s"Unsupported File Format found: ${f.toString}.")
```

Comment:
usually `other` is used for the variable name (in Spark, at least)

Reply:
also, either `FileFormat` or `file format`

Reply:
thanks @imback82 , done